src/CommandLine.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Scanner;
+
+public class CommandLine{
+    public static void main(String[] args){
+        Scanner sc = new Scanner(System.in);
+		int input = getInput(sc);
+    
+        if(input == 1){
+            String file = getFileInput(sc);
+            System.out.println("File imported: " + file);
+        }else if(input == 2){
+			int[] integers = getIntegerInput(sc);
```

Comment:
looks good over all, but some of the indentation is a little off. These lines have too many indents, but I'll fix it when I integrate it into our engine client. Thanks!